proto/config/storage.proto

```diff
@@ -208,6 +208,12 @@ message Volume {
   string displayName = 7;  // Optional friendly name echo'ed in info message
   bool clear_text = 8;     // Flag to indicate the volume encryption needed or not
   Target target = 9;       // How the volume will be presented to the instance. At the moment only the AppCustom is used
+
+  // To inform the edge-node if the device receiving this Volume is
+  // responsible for volume creation, convert PVC, or not
+  string DesignatedNodeID = 10;
+  // replicated to devices with UUID array, number range from 0 to 2
```

Comment:
this 'repeated string' below is the UUID array.

Reply:
I am confused. Is it a UUID? or a number range from 0 to 2 per the comment?

Reply:
we meant the size of the array is a number, I think it should be 3, to list all the UUIDs needed for this VI for replication. Unless this VI does not need to be replicated by longhorn, then it can be zero size of array.

Reply:
The API shouldn't have a max number. And the comment about number range is confusing.

But are we planning to implement controlling the replicas in the next three months or so?
If not, why do we need any of these storage-specific items.

We can just have the controller specify the members of the cluster (message EdgeNodeCluster).
We can add more detailed control as those needs arise and we know better what should be done in the controller vs. by the EVE cluster autonomously.

Can't the cluster elect who should create the volume/PVC?

Reply:
Likely not in first three months. So it's ok to remove this array. We assume everything is replicated to the only 3 nodes.
Have cluster elect works for now, but i doubt will work properly later. since the VIs can be downloaded first, and Apps can be downloaded later, if we have more than 3 nodes invovled, they better be from the consistent configures from the controller. The schemes should work for now and three months or more later.

Reply:
comment and the associated field replica_node_id have been removed